components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthUtil.java

```diff
@@ -1192,32 +1184,6 @@ public static String getUsername(String tenantDomain) throws IdentityApplication
         return username;
     }
 
-    /**
```

Comment:
This method introduced with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2085/files
Previously there was an exception when `userId = authorizedUser.getUserId()`. 
But with the improved logic inside `authorizedUser.getUserId()`, the federated organization SSO user ID resolving is handled, hence this exception will not occurred, hence we can clean up this code.